modules/docs/arrow-docs/docs/docs/integrations/rx2/README.md

```diff
@@ -85,7 +85,7 @@ subject.value()
 
 ### Observable comprehensions
 
-The library provides instances of [`MonadError`]({{ '/docs/typeclasses/monaderror' | relative_url }}) and [`MonadDefer`]({{ '/docs/effects/monaddefer' | relative_url }}).
+The library provides instances of [MonadError]({{ '/docs/typeclasses/monaderror' | relative_url }}) and [`MonadDefer`]({{ '/docs/effects/monaddefer' | relative_url }}).
```

Comment:
Backticks for MonadError.

Reply:
rx2??

Reply:
These -> `

Reply:
Oh I see. I shouldn't have edited the file in the rx2. It was a mistake